src/payment/strategies/googlepay/googlepay.ts

```diff
@@ -48,7 +48,7 @@ export interface TokenizePayload {
         lastTwo?: string;
     };
     description?: string;
-    type: TokenizeType;
+    type: TokenizeType | string;
```

Comment:
doesn't this affect all other google pay implementations? 🤔

Reply:
The other Google Pay implementations use this TokenizeType. But, in this case, it's not necessary. So I need it to be a simple string.